### PR TITLE
Vale initial work

### PR DESCRIPTION
**Pull request by jeroenleenarts**
### 🎯 Goal

_Describe why we are making this change_

### 🛠 Implementation details

_Describe the implementation_

### 🎨 UI Changes

_Add relevant screenshots_

| Before | After |
| --- | --- |
| img | img |

_Add relevant videos_

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="" controls="controls" muted="muted" />
</td>
<td>
<video src="" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

_Explain how this change can be tested (or why it can't be tested)_

_Provide a patch below if it is necessary for testing_

<details>

<summary>Provide the patch summary here</summary>

```
Provide the patch code here
```

</details>

#### General
- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] Assigned a person / code owner group (required)
- [ ] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [ ] PR targets the `develop` branch
- [ ] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_

### CODE REVIEW

File: docusaurus/docs/Android/03-ui/04-message-components/01-message-list-screen.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ The `MessageListFragment` contains these three inner components:
 * [`MessageInputView`](04-message-input.mdx): Handles the message input, as well as attachments and message actions like editing and replying.
 
 :::note
-Fragments and Activities representing self-contained screens are easy to use. They allow you to explore the SDK's features in a breeze, however, they offer limited customization.
+Fragments and Activities representing self-contained screens are easy to use. They allow you to explore the SDKs features in a breeze, however, they offer limited customization.
```
Comment (filbabic) [inline docusaurus/docs/Android/03-ui/04-message-components/01-message-list-screen.mdx:18]:
```suggestion
Fragments and Activities representing self-contained screens are easy to use. They allow you to explore the features of our SDK in a breeze, however, they offer limited customization.
```

**Comment by jeroenleenarts** [comment]
Check the 3rd step in the checks.

image

**Comment by JcMinarro** [comment]
@jeroenleenarts PRs need to be created to be merged over `develop` branch
Let me update the base branch

**Review by filbabic — APPROVED** [review]
Really really nice, thanks!

**Pull request merged**